ssl/t1_lib.c

```diff
@@ -2550,6 +2549,10 @@ static int ssl_scan_serverhello_tlsext(SSL *s, PACKET *pkt, int *al)
                 return 0;
             }
             s->s3->alpn_selected_len = len;
+#ifndef OPENSSL_NO_NEXTPROTONEG
```

Comment:
Ugh... looks as though my branch was not up-to-date with master, and the other ALPN changes are not in there? :-/ Let me double-check this and update the PR.